x-pack/test/alerting_api_integration/security_and_spaces/tests/actions/builtin_action_types/servicenow_access_token.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { generateKeyPairSync } from 'crypto';
+import expect from '@kbn/expect';
+import httpProxy from 'http-proxy';
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+import { getHttpProxyServer } from '../../../../common/lib/get_proxy_server';
+import {
+  ExternalServiceSimulator,
+  getExternalServiceSimulatorPath,
+} from '../../../../common/fixtures/plugins/actions_simulators/server/plugin';
+
+// eslint-disable-next-line import/no-default-export
+export default function serviceNowAccessTokenTest({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const kibanaServer = getService('kibanaServer');
+
+  describe('get servicenow access token', () => {
+    let servicenowSimulatorURL: string = '';
+    let proxyServer: httpProxy | undefined;
+    const configService = getService('config');
+
+    // need to wait for kibanaServer to settle ...
+    before(async () => {
+      servicenowSimulatorURL = kibanaServer.resolveUrl(
+        getExternalServiceSimulatorPath(ExternalServiceSimulator.SERVICENOW)
+      );
+      proxyServer = await getHttpProxyServer(
+        kibanaServer.resolveUrl('/'),
+        configService.get('kbnTestServer.serverArgs'),
+        () => {}
+      );
+    });
+
+    after(() => {
+      if (proxyServer) {
+        proxyServer.close();
+      }
+    });
+
+    it('should return 200 when requesting an access token with OAuth credentials', async () => {
+      const { privateKey } = generateKeyPairSync('rsa', {
```

Comment:
Given that this is a somehow expensive task to have on the CI, can we use a fixed one? Is the key from here `packages/kbn-dev-utils/src/certs.ts` suitable for our case? You can import it as `import { KBN_KEY_PATH } from '@kbn/dev-utils'`

Reply:
Great idea! I didn't know about that so thank you for pointing it out. Updated in this commit: https://github.com/elastic/kibana/pull/131171/commits/d3422a09f2f0429614ce83af33bf1da0ad21e085